docs/tutorials/stateful-application/cassandra.md

```diff
@@ -1,374 +1,134 @@
-kubectl scale rc cassandra --replicas=4
+To avoid these errors add the flags `--memory 5120` and `--cpus=4` to `minikube start`
 
-# delete the replication controller
-kubectl delete rc cassandra
-
-#
-# Create a DaemonSet to place a cassandra node on each kubernetes node
-#
-
-kubectl create -f cassandra/cassandra-daemonset.yaml --validate=false
-
-# resource cleanup
-kubectl delete service -l app=cassandra
-kubectl delete daemonset cassandra
 ```
+minikube start --memory 5120 --cpus=4
+``` 
+* To download [cassandra-service.yaml](docs/tutorials/stateful-application/cassandra-service.yaml) and [cassandra-statefulset.yaml](/docs/tutorials/stateful-application/cassandra-statefulset.yaml)
+{% endcapture %}
 
-## Step 1: Create a Cassandra Headless Service
-
-A Kubernetes _[Service](/docs/user-guide/services)_ describes a set of
-[_Pods_](/docs/user-guide/pods) that perform the same task. In
-Kubernetes, the atomic unit of an application is a Pod: one or more containers
-that _must_ be scheduled onto the same host.
+{% capture lessoncontent %}
+## Creating a Cassandra Headless Service
+A Kubernetes [Service](/docs/user-guide/services) describes a set of [Pods](/docs/user-guide/pods) that perform the same task. 
 
-The Service is used for DNS lookups between Cassandra Pods, and Cassandra clients
-within the Kubernetes Cluster.
+The following `Service` is used for DNS lookups between Cassandra pods and clients within the Kubernetes Cluster.
 
-Here is the service description:
+1. `cd` to the folder you saved the .yaml files.
+2. Create a Service to track all Cassandra StatefulSet Nodes from the following .yaml file:
 
-<!-- BEGIN MUNGE: EXAMPLE cassandra-service.yaml -->
-
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  labels:
-    app: cassandra
-  name: cassandra
-spec:
-  clusterIP: None
-  ports:
-    - port: 9042
-  selector:
-    app: cassandra
+```shell
+kubectl create -f cassandra-service.yaml
 ```
 
-[Download example](https://raw.githubusercontent.com/kubernetes/examples/master/cassandra-service.yaml)
-<!-- END MUNGE: EXAMPLE cassandra-service.yaml -->
+{% include code.html language="yaml" file="cassandra-service.yaml" ghlink="/docs/tutorials/stateful-application/cassandra-service.yaml" %}
 
-Create the service for the StatefulSet:
+### Validating (optional)
 
+* Get the Cassandra `Service`.
 
-```console
-$ kubectl create -f cassandra/cassandra-service.yaml
+```shell
+kubectl get svc cassandra
 ```
 
-The following command shows if the service has been created.
-
-```console
-$ kubectl get svc cassandra
-```
-
-The response should be like:
+The response should be
 
 ```console
 NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
 cassandra   None         <none>        9042/TCP   45s
 ```
 
-If an error is returned the service create failed.
+If anything else returns, the service was not successfully created. Check out [Debug Services](/docs/tasks/debug-application-cluster/debug-service/) for common issues.
```

Comment:
Nitpick.  "Check out" is difficult to parse for someone who doesn't speak english as a first language.  It's especially difficult since "Check out" can mean "check out code from git".  I suggest using the word "Read" instead of the phrase "check out".  This phrase is used in other parts of the doc too.